pkg/apiserver/admission/chain_builder.go

```diff
@@ -210,11 +210,15 @@ func NewAdmissionChains(
 		admissionPluginConfigFilename = tempFile.Name()
 	}
 
-	admissionPluginNames := openshiftAdmissionControlPlugins
-	admissionPluginNames = fixupAdmissionPlugins(admissionPluginNames)
```

Comment:
open a bugzilla for this.  we should fix it during migration.

Reply:
Created https://jira.coreos.com/browse/MSTR-214